### PR TITLE
Update Main.hx

### PR DESCRIPTION
**Pull request by myloveinapril**
I ADDED BACK THE FUCKING HAXEFLIXEL INTRO HAHAHHAhAHAHA!!!

### CODE REVIEW

**Comment by tposejank** [comment]
No

**Comment by SonaIsCool** [comment]
> No

shit yourself

**Comment by SrtHero278** [comment]
why'd you force on fullscreen?

File: source/Main.hx [hunk 1]
```diff
@@ -18,8 +18,8 @@ class Main extends Sprite
 	var initialState:Class<FlxState> = TitleState; // The FlxState the game starts with.
 	var zoom:Float = -1; // If -1, zoom is automatically calculated to fit the window dimensions.
 	var framerate:Int = 60; // How many frames per second the game should run at.
-	var skipSplash:Bool = true; // Whether to skip the flixel splash screen that appears in release mode.
-	var startFullscreen:Bool = false; // Whether to start the game in fullscreen on desktop targets
+	var skipSplash:Bool = false; // Whether to skip the flixel splash screen that appears in release mode.
+	var startFullscreen:Bool = true; // Whether to start the game in fullscreen on desktop targets
```
Comment (mayo78) [inline source/Main.hx:22]: wtf this forces fullscreen on startup you should be banned and murdered (irl)

Comment (JohanDaHooman) [reply source/Main.hx:22]: imagine forcing fullscreen on startup + ratio + 152.141.239.74

**Comment by trayfellow** [comment]
noooo! the force full screen .. . on... ! ! 

.

File: source/Main.hx (hunk 1 above)
Comment (tposejank) [reply source/Main.hx:22]: 574.851.152.68

**Comment by frantastic24** [comment]
They beating your ass in the reviews 
Anyways nice pointless PR

File: source/Main.hx (hunk 1 above)
Comment (SonaIsCool) [reply source/Main.hx:22]: alright hold on im ognna change it

**Comment by Stilic** [comment]
no

**Comment by SonaIsCool** [comment]
125.234.23.162

**Comment by MayoALT** [comment]
> 125.234.23.162

ooooooooooooo  u doxxed someone you will get the death penalty with no trial!!!!!!!!!!!!!!!!

**Comment by SonaIsCool** [comment]
tu madre

**Comment by lemz1** [comment]
127.0.0.1

**Comment by tposejank** [comment]
> tu madre

https://youtu.be/R8RnYvRXlvA

**Comment by SonaIsCool** [comment]
best argument thats ever happened
time to close this cus shadow marios probably gonna fucking murder me

**Pull request closed without merging**